apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -106,6 +107,31 @@ const useStyles = makeStyles((theme) => ({
     paddingLeft: '8px',
     paddingRight: '8px',
   },
+  barContainer: {
+    position: 'relative',
+  },
+  bar: {
+    border: '1px solid',
+    borderColor: '#D9D9D9',
+    height: '57%',
```

Comment:
Looks fine in the pics, just wondering if the remaining `43%` can be divided equally (i.e. 21.5%) on top and bottom.

Reply:
I'm thinking pixels here, not sure if this matters for percent.

Reply:
Ok, 1em might look better.

Reply:
Percent should be fine. I was just thinking of how changing it into 42% or 44% to make the remaining space dividable by 2 (if it matters).